scripts/parse.py

```diff
@@ -102,7 +102,8 @@ def main():
         ]
 
     for source, revision, url, schema in url_list:
-        req = requests.get(url)
+        user_agent = {'User-agent': 'Mozilla/5.0'}
+        req = requests.get(url,headers = user_agent)
```

Comment:
`req = requests.get(url, headers=user_agent)`

[flake8](http://flake8.pycqa.org/en/latest/) is our friend.